feat: comma separated list of content_type

dopry opened the pull request:
applies to #249

zerolab commented on grapple/types/pages.py:
```diff
@@ -284,10 +286,17 @@ def resolve_pages(self, info, **kwargs):
                 pages = pages.filter(
-                    content_type__app_label=app_label, content_type__model=model
+                    content_type__app_label__in=app_labels,
+                    content_type__model__in=models,
```
This doesn't feel right, we need to check pairs of app label/model, rather than two lists. suggest constructing a set of OR-ed `Q()` objects

dopry replied:
good catch.  Updated and rebased.

zerolab reviewed (commented):
The changes look good to me now.

I do feel we need to update the docs https://wagtail-grapple.readthedocs.io/en/latest/general-usage/graphql-types.html?highlight=contentType#pageinterface to mention this, otherwise it stays hidden

dopry commented:
I would argue that it is already hidden. In the docs, contentType only says String. It doesn't indicate that you need to use app.Model notation. I don't necessarily I want to get into writing a lot of docs that is not my forte. I did update the description so it is apparent when working in graphiql.

dopry commented:
I added a quick comment in the doc...  I also updated the docs for my ancestor/parent PR. It would be nice if the type docs were just derived from the graphql schema so contributors didn't need to update them in two places...

zerolab commented:
> It would be nice if the type docs were just derived from the graphql schema so contributors didn't need to update them in two places

I agree. It is always a trade-off between working on a feature or working on thing around (like docs, tooling, etc), and the time available. If you have the time and want to tinker, please do :)

dopry commented:
Thanks for getting this merged so quickly! I really appreciate it. I, unfortunately, don't have time to pursue the documentation generation endeavor. I need to be focused on my current project for the next month or two and have other open source projects that consume my leisure time. I do expect you'll be seeing me around the grapple/headless-preview/admin client space for the duration of this project.
